http-serverless/README.md

```diff
@@ -0,0 +1,5 @@
+# Micronaut HTTP Serverless
```

Comment:
we need to rename this to something else, Serverless has a broad meaning and many different implementations, this is just one. Perhaps `http-raw-serverless` or something, WDYT @yawkat

Reply:
Fair point, @graemerocher. Since this module adds enables Plain Old Java Applications(_POJA_) to use Micronaut Http Stack, I have now renamed it to **http-poja**.